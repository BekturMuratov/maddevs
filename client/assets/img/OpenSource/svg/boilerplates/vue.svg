<svg xmlns="http://www.w3.org/2000/svg" width="336" height="125" fill="none" xmlns:v="https://vecta.io/nano"><g clip-path="url(#A)"><path d="M206.784.001l-38.797 67.203L129.191.001H-.008l167.996 290.997L335.984.001h-129.2z" fill="#41b883"/><path d="M206.785.001l-38.797 67.203L129.191.001h-62l100.798 174.595L268.786.001h-62.001z" fill="#34495e"/></g><defs><clipPath id="A"><path fill="#fff" d="M0 0h336v291H0z"/></clipPath></defs></svg>