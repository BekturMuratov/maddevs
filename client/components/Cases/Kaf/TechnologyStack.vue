<template>
  <section class="container_regular">
    <h2 class="case_title_h2 m-24_bottom">Technology stack</h2>
    <!-- <ListTechnologies class="m-96_bottom media-m-48_bottom">
      <ListTechnologiesItem
        v-for="(technologiesItem, i) in technologiesList"
        :key="i"
        :tech-name="technologiesItem.techName"
        :class-name="technologiesItem.className"
      />
    </ListTechnologies> -->
  </section>
</template>

<script>
export default {
  name: 'TechnologyStack',
}
</script>
