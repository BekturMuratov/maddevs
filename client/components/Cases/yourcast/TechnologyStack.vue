<template>
  <section class="container_regular">
    <h2 class="case_title_h2 m-24_bottom">
      Technology stack
    </h2>
  </section>
</template>

<script>
export default {
  name: 'TechnologyStack',
}
</script>
