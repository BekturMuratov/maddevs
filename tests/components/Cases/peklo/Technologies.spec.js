import TechnologyStack from '@/components/Cases/peklo/Technologies'
import { render, screen } from '@testing-library/vue'

describe('Technologies component', () => {
  it('should render correctly', () => {
    const { container } = render(TechnologyStack)
    expect(container).toMatchSnapshot()
  })

  it('title should include classname', () => {
    render(TechnologyStack)
    expect(screen.getByText(/Technology stack/i).className).toContain('case_title_h2')
  })
})
