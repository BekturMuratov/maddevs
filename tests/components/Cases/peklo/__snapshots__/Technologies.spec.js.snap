// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Technologies component should render correctly 1`] = `
<div>
  <section
    class="container_regular"
  >
    <h2
      class="case_title_h2 m-24_bottom media-m-12_bottom"
    >
      
    Technology stack
  
    </h2>
     
    <ul
      class="case_list-technologies m-96_bottom media-m-48_bottom"
    >
      <li
        class="case_technologies-item"
      >
        <img
          alt="Golang"
          class="img_lazy case_technologies-item__tech-icon case_technologies-item__go go"
          data-src="[object Object]"
          height="40"
          width="40"
        />
         
        <p
          class="case_technologies-item__tech-name"
        >
          
    Golang
  
        </p>
      </li>
      <li
        class="case_technologies-item"
      >
        <img
          alt="PostgreSQL"
          class="img_lazy case_technologies-item__tech-icon case_technologies-item__postgresql postgresql"
          data-src="[object Object]"
          height="40"
          width="40"
        />
         
        <p
          class="case_technologies-item__tech-name"
        >
          
    PostgreSQL
  
        </p>
      </li>
      <li
        class="case_technologies-item"
      >
        <img
          alt="Docker"
          class="img_lazy case_technologies-item__tech-icon case_technologies-item__docker docker"
          data-src="[object Object]"
          height="40"
          width="40"
        />
         
        <p
          class="case_technologies-item__tech-name"
        >
          
    Docker
  
        </p>
      </li>
      <li
        class="case_technologies-item"
      >
        <img
          alt="Gitlab"
          class="img_lazy case_technologies-item__tech-icon case_technologies-item__gitlab gitlab"
          data-src="[object Object]"
          height="40"
          width="40"
        />
         
        <p
          class="case_technologies-item__tech-name"
        >
          
    Gitlab
  
        </p>
      </li>
      <li
        class="case_technologies-item"
      >
        <img
          alt="Ruby"
          class="img_lazy case_technologies-item__tech-icon case_technologies-item__ruby ruby"
          data-src="[object Object]"
          height="40"
          width="40"
        />
         
        <p
          class="case_technologies-item__tech-name"
        >
          
    Ruby
  
        </p>
      </li>
      <li
        class="case_technologies-item"
      >
        <img
          alt="Vue.js"
          class="img_lazy case_technologies-item__tech-icon case_technologies-item__vue vue"
          data-src="[object Object]"
          height="40"
          width="40"
        />
         
        <p
          class="case_technologies-item__tech-name"
        >
          
    Vue.js
  
        </p>
      </li>
    </ul>
  </section>
</div>
`;
